ask-sdk-core/ask_sdk_core/serialize.py

```diff
@@ -67,8 +67,8 @@ def serialize(self, obj):
         If obj is list, serialize each element in the list.
         If obj is dict, return the dict with serialized values.
         If obj is ask sdk model, return the dict with keys resolved
```

Comment:
Since this `serialize` method would work for other than `ask sdk model`, I would like that functionality to be mentioned in the docstring. So ideally, something like, `if obj is ask sdk model then ... happens. if obj is a generic class instance, then ... happens` kind of information, What do you think? 

Similar information to be added in the `deserialize` method docstring

Reply:
There isn't really a difference if the serialized object is a ASK model or not, the function always behaves the same. That said, you're the maintainer, so I can do what you feel is best. I don't feel knowledgeable enough to write correct docstrings here, I don't have the full picture. I think you should be able to update this PR as well, so please feel free to do so.